plugins/GeoIP/class.geoip.import.php

```diff
@@ -0,0 +1,683 @@
+<?php
+
+class GeoipImport {
+
+    private static $csvDownloadURL = "http://geolite.maxmind.com/download/geoip/database/GeoLite2-City-CSV.zip";
```

Comment:
Variables need docblocks

/**
 * URL to download IP database
 * @var string
 */